test-framework/common/src/main/java/io/quarkus/test/common/JarLauncher.java

```diff
@@ -39,6 +40,7 @@ private JarLauncher(Path jarPath, Config config) {
                 config.getValue("quarkus.http.test-port", OptionalInt.class).orElse(DEFAULT_PORT),
                 config.getValue("quarkus.http.test-ssl-port", OptionalInt.class).orElse(DEFAULT_HTTPS_PORT),
                 config.getValue("quarkus.test.jar-wait-time", OptionalLong.class).orElse(DEFAULT_JAR_WAIT_TIME),
+                config.getOptionalValue("quarkus.test.argLine", String.class).orElse(null),
```

Comment:
So this is something you expect to set in `application.properties`?

Reply:
I'd expect it to be set exactly how `quarkus.test.native-image-profile` is set, when running tests.

In my case, I use the failsafe plugin systemPropertyVariables and the JarLauncher picks those up.

But it only populates the properties it knows to look for, rather than all the properties you set, so this adds the property to be known to look for.

Reply:
Great, just making sure we are on the same page.

Reply:
Should I do anything about the breaking tests in the automated builds? They fail for me locally as well, before and after the change. Will that hold up this PR?

Reply:
@glefloch any idea what's going with the Gradle tests?

Reply:
Those failures have been fixed but it looks like the branch is old (it tries to download kotlin lib in 1.3.72 and we migrated all tests to use 1.4.2). @bvarner could you rebase your branch with main?

Reply:
I can do that. I made the change off 1.13..

Reply:
Please base the PR off of `main`. 
If we decide to backport this to `1.13`, we can do it later on

Reply:
Got it. The rebase is pretty hairy. lots of renaming and quite a few conflicts (I assume from backports).... 

Would it be better to just close this PR and issue a new one from main?

Reply:
Yeah. I'm going to open a different PR for this (way easier than trying to comb through the conflicts)

Reply:
Whatever works best for you